### PR TITLE
scripts: Add support for running http_server-cpp

### PR DESCRIPTION
**Pull request by b1shtream**

### CODE REVIEW

**Comment by razvand** [comment]
@chococandy63 , what is this issue for?

File: scripts/run.yaml [hunk 1]
```diff
@@ -14,3 +14,8 @@ runs:
     command: /usr/bin/sqlite3 
     memory: 64
     networking: False
+  - name: http_server
```
Comment (razvand) [inline scripts/run.yaml]:
```suggestion
  - name: http_server-cpp
```

To differentiate it from other servers.

**Review by razvand — CHANGES_REQUESTED** [review]
Use `run.yaml:` instead of `scripts:` as prefix for commit message.

**Comment by razvand** [comment]
10 HP

**Comment by b1shtream** [comment]
@razvand I have made all the changes, kindly review them again.

**Review by razvand — APPROVED** [review]
Reviewed-by: Razvan Deaconescu <razvand@unikraft.io>
Approved-by: Razvan Deaconescu <razvand@unikraft.io>

**Pull request merged**